db/statistics.go

```diff
@@ -28,13 +28,13 @@ func WriteValidatorStatisticsForDay(day uint64) error {
 
 	logger.Infof("exporting statistics for day %v (epoch %v to %v)", day, firstEpoch, lastEpoch)
 
-	latestDbEpoch, err := GetLatestEpoch()
+	finalizedCount, err := CountFinalizedEpochs(firstEpoch, lastEpoch)
 	if err != nil {
 		return err
 	}
 
-	if lastEpoch > latestDbEpoch {
-		return fmt.Errorf("delaying statistics export as epoch %v has not yet been indexed. LatestDB: %v", lastEpoch, latestDbEpoch)
+	if finalizedCount < epochsPerDay {
+		return fmt.Errorf("delaying statistics export as not all epochs for this day are finalized. LatestDB: %v", day)
```

Comment:
**Minor:**
The use of `day` here is wrong, is it not? Maybe the last part can be removed?
```suggestion
		return fmt.Errorf("delaying statistics export as not all epochs for this day are finalized.")
```

Furthermore, you could add something like "(%d of %d)" with the parameters `finalizedCount` and `epochsPerDay`.

Reply:
fixed error